docs/install/knative-with-operators.md

```diff
@@ -127,19 +127,20 @@ Information about Knative Eventing Operator releases can be found on the [Releas
 
 __From releases__:
 
-Replace \<version\> with the latest version or the version you would like to install, and run the following command to
-install Knative Eventing Operator:
+Run the following command to install Knative Eventing Operator:
+
 
 ```
-kubectl apply -f https://github.com/knative/eventing-operator/releases/download/<version>/eventing-operator.yaml
+kubectl apply --filename {{< artifact repo="eventing-operator" file="eventing-operator.yaml" >}}
 ```
 
 __From source code__:
 
 You can also install Knative Operators from source using `ko`.
 
-- Install the [ko]((https://github.com/google/ko)) build tool.
-- Install the operator in the root directory of the source using the following command:
+1. Install the [ko]((https://github.com/google/ko)) build tool.
```

Comment:
This is another error I brought in with the  double brackets.
It should be [ko](https://github.com/google/ko)